lib/react_client.dart

```diff
@@ -167,23 +171,20 @@ dynamic _convertArgsToChildren(List childrenArgs) {
 @JS('Object.keys')
 external List<String> _objectKeys(Object object);
 
-InteropContextValue _jsifyContext(Map<String, dynamic> context) {
-  var interopContext = new InteropContextValue();
-  context.forEach((key, value) {
-    setProperty(interopContext, key, new ReactDartContextInternal(value));
-  });
-
+InteropContextValue _jsifyContext(dynamic context) {
+  dynamic interopContext = new InteropContextValue();
+  setProperty(interopContext, '___internal_dart_context_value___',
+      new ReactDartContextInternal(context));
   return interopContext;
 }
 
-Map<String, dynamic> _unjsifyContext(InteropContextValue interopContext) {
-  // TODO consider using `contextKeys` for this if perf of objectKeys is bad.
-  return new Map.fromIterable(_objectKeys(interopContext), value: (key) {
-    ReactDartContextInternal internal = getProperty(interopContext, key);
-    return internal?.value;
-  });
+dynamic _unjsifyContext(dynamic interopContext) {
+  return getProperty(interopContext, '___internal_dart_context_value___')
+      ?.value;
 }
 
+final _emptyJsMap = newObject();
```

Comment:
Can we move this directly underneath the deprecated public `emptyJsMap` - just so its easier to see what happened / the reason for the deprecation of the public one?